src/main/asyncapi/network_survey_messaging.yaml

```diff
@@ -221,6 +224,32 @@ components:
       payload:
         $ref: '#/components/schemas/lteNasPayload'
 
+    umtsNas:
+      name: UmtsNas
+      title: UMTS NAS Message
+      summary: Represents a raw UMTS NAS message sent Over The Air (OTA) between an UMTS uNodeB and an UMTS UE.
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/umtsNasPayload'
+
+    umtsNasDsds:
+      name: UmtsNasDsds
+      title: UMTS NAS DSDS Message
+      summary: Represents a raw UMTS NAS DSDS message sent Over The Air (OTA) between an UMTS uNodeB and an UMTS UE.
```

Comment:
Switch to "**a** UMTS NodeB and **a** UMTS UE"

Reply:
Updated.